inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/add.cpp

```diff
@@ -10,24 +10,33 @@
 #include "common_test_utils/test_constants.hpp"
 
 using namespace LayerTestsDefinitions;
-
+using namespace AddTestDefinitions;
 namespace {
 
-const std::vector<InferenceEngine::Precision> netPrecisions = {
-        InferenceEngine::Precision::FP32,
-        InferenceEngine::Precision::FP16
+std::vector<std::vector<std::vector<size_t>>> inputShapes = { {std::vector<std::size_t>({1, 30})} };
```

Comment:
As in test for Multiply layer we can also add more input shapes here

Reply:
Sure, I'll add additional shapes, I simply left the "original" shapes of the test